Procfile

```diff
@@ -0,0 +1,2 @@
+web: gunicorn realworld.wsgi
+release: python manage.py migrate
```

Comment:
I generally recommend against automatic migrations since they so often go wrong, but it's probably fine for this app!